Makefile

```diff
@@ -11,7 +11,7 @@ check: static-analysis unit-tests integration-tests coding-standards
 
 static-analysis: vendor
 	vendor/bin/parallel-lint $(sources)
-	vendor/bin/phpstan analyse --level=7 $(sources)
+	vendor/bin/phpstan analyse --level=7 --configuration=phpstan.neon $(sources)
```

Comment:
Just so you know:

> If you do not provide config file explicitly, PHPStan will look for files named phpstan.neon or phpstan.neon.dist in current directory.

(https://github.com/phpstan/phpstan#configuration)

Reply:
Ah, it turns out that the [auto-discovery of `phpstan.neon`](https://github.com/phpstan/phpstan/issues/237) is a [recently merged feature](https://github.com/phpstan/phpstan/pull/509), that is not part of a released version yet. It looks like it will be released as part of the [upcoming 0.10 release](https://github.com/phpstan/phpstan/milestone/8).

It is, however, merged into the master branch, which is why the repository's README documents the behavior.

It looks like we have to wait for a new release, and in the meanwhile specify the configuration file like you did here :+1: